File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.cpp
class Solution {
public:
    int countPairs(vector<int>& n, int tgt) {
        int cnt =0;
        for( int i=0;i<n.size();++i){
            for( int j=i+1;j<n.size();++j){
                if(n[i]+n[j]<tgt)++cnt;
            }
        }
        return cnt;
    }
};

File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.py
class Solution:
    def countPairs(self, n: List[int], t: int) -> int:
        cnt =0
        for i in range( len(n)):
            for j in range (i+1,len(n)):
                if n[i]+n[j] < t:
                    cnt+=1
        return cnt